packages/react-devtools/package.json

```diff
@@ -1,7 +1,9 @@
 {
   "name": "react-devtools",
+  "productName": "React Developer Tools",
   "version": "2.0.12",
   "description": "Use react-devtools outside of the browser",
+  "main": "./packages/react-devtools/app.js",
```

Comment:
Is this still the right path? I'm actually thinking we intentionally have a different entry point because we want people to `require('react-devtools')` and get [the client](https://github.com/facebook/react-devtools/blob/master/packages/react-devtools/index.js) running.